projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdf.java

```diff
@@ -0,0 +1,248 @@
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import com.itextpdf.licensekey.LicenseKey;
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.html2pdf.*;
+
+import java.io.BufferedReader;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+import noloan.R;
+
+public class LawsuitPdf extends AppCompatActivity { // TODO: Add logs to class functions.
+
+
+    private static final String TAG = "Lawsuit";
+
+    // Folder configurations:
+    // TODO: Check if preferable to move these to an external folder configuration file.
+    private static String appName = "";                                             // Get App's name as main dir.
+    private static final String lawsuitPath = "/Lawsuits/";                         // General lawsuits dir.
+    private static String templatePath = lawsuitPath + "/Templates/template.xhtml"; // Lawsuit Template's dir & template's path.
+    private static final String lawsuitOutput = lawsuitPath;                        // Lawsuits output dir.
+
+    // Date & Time formats: (Used for files names)
+    // TODO: Check if this should be as part of a "File manager" class.
+    private static final String DATE_TIME_FORMAT = "dd-M-yyyy hh-mm-ss";
+    private static final String DATE_FORMAT = "dd-M-yyyy";
+    private static final String TIME_ZONE = "Asia/Jerusalem";
+    private static SimpleDateFormat dateAndTimeFormatter;
+    private static SimpleDateFormat dateFormatter;
+
+    // Template's inner text (been read on create from template's file):
+    private static String templateContent = "";
+
+// ------------------------------- Lawsuit Form Fields ------------------------------- //
+//  TODO: Check if preferable to move this to an external file holding constant String values of lawsuit form.
+    private String spamType = "הודעה אלקטרונית";        // Type: SMS/Phone call/Email.
+    private String claimAmount = "סכום תביעה";
+    private Date receivedSpamDate;                    // Date received the spam.
+    private Date currentDate;                         // Current date of lawsuit creation (used for lawsuit signature).
+    private boolean sentHaser = true;                 // Indicates whether user has sent "haser" or not.
+    private boolean sentMoreThanFiveLawsuits = false; // Indicates whether user has sent more then 5 lawsuits already in the past year or not.
+    private String moreThanFiveLawsuits = sentMoreThanFiveLawsuits ? "הגיש" : "לא הגיש/ו";
+    private String claimCase = sentHaser ? "לאחר שהתובע חזר בו מהסכמתו לקבלת דבר/י פרסומת מהנתבע/ים, על-ידי משלוח הודעת סירוב כהגדרתה בחוק" : "למרות שח\"מ לא נתן את הסכמתו המפורשת מראש לקבלת דבר/י הפרסומת";
```

Comment:
`claimCase` is evaluated when the class is instantiated, and so is `sentHaser`, which is set to `true`, so wouldn't `claimCase` always get the same value?